spring-ai-docs/src/main/antora/modules/ROOT/pages/upgrade-notes.adoc

```diff
@@ -5,6 +5,11 @@
 
 * The configuration prefix for the Chroma Vector Store has been changes from `spring.ai.vectorstore.chroma.store` to `spring.ai.vectorstore.chroma` in order to align with the naming conventions of other vector stores.
 
+* The default value of the `initialize-schema` property on vector stores capable of initializing a schema is now set to `false`.
+This implies that the applications now need to explicitly opt-in for schema initialization on supported vector stores, if the schema is expected to be created at application startup.
+Not all vector stores support this property.
```

Comment:
we should list the one's that don't so they don't have to search.  Also, I don't think we are very clear on this in the ref docs itself, will need to double check.

Reply:
My quick check of those that don't are 

* `Cassandra` - I have a feeling though it should.  There is a property `disallowSchemaChanges` that seems like it is performing the same task as `initializeSchema`
* `Gemfire` - The implementation of `afterPropertiesSet` looks problematic, which is a separate issue, but seems like we would need to update that method to work off `initializeSchema`  @geetrawat can you comment. 
* `Hana` 
* `OpenSearch` implements `InitializingBean` with an index being created if it doesn't exist.  Should be retrofitted to use `initializeSchema`
* `Pinecone`
* `Typesense` implements  `InitializingBean` with an collection being created if it doesn't exist.  Should be retrofitted to use `initializeSchema`